Apps/Sandcastle/CesiumSandcastle.js

```diff
@@ -377,7 +378,7 @@ require({
                     errorLines.push(line);
                 }
             }
-        }
+        }/*eslint-enable new-cap*/
```

Comment:
I will add a "comments" section that includes a sub-section for ESLint configuration comments